includes/rest-api.php

```diff
@@ -600,8 +600,11 @@ function check_post_types_permissions() {
 	);
 
 	foreach ( $types as $type ) {
-		$caps                  = $type->cap;
-		$response['can_get'][] = $type->name;
+		$caps = $type->cap;
+
+		if ( current_user_can( $caps->edit_posts ) ) {
+			$response['can_post'][] = $type->name;
```

Comment:
@peterwilsoncc should this be `$response['can_get'][] = $type->name;` instead?

Reply:
> should this be `$response['can_get'][] = $type->name;` instead?

Yes, it should be. Sorry about that and thanks for picking up my silly moment.

Reply:
Thank you and no worries. Updated! 😄